Gemfile

```
@@ -82,6 +82,7 @@ group :development do
   gem 'derailed_benchmarks', '~> 1.8'
   gem 'guard-rspec', require: false
   gem 'irb'
+  gem 'letter_opener', '~> 1.7'
```

I expect it wouldn't have a practical impact, but should we specify the latest minor?

```suggestion
  gem 'letter_opener', '~> 1.8'
```

---

will do